presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformUncorrelatedInPredicateSubqueryToSemiJoin.java

```diff
@@ -253,7 +254,8 @@ protected PlanNode rewriteApply(ApplyNode node)
                         valueList,
                         semiJoinSymbol.get(),
                         Optional.empty(),
-                        Optional.empty()
+                        Optional.empty(),
+                        DISTRIBUTED
```

Comment:
Doesn't matter because it gets rewritten later, but I'll do as you suggest and make the join method optional so that it's clear that it's not really being set.